Documentation/Books/Manual/Foxx/Reference/Routers/Endpoints.md

```diff
@@ -482,3 +481,146 @@ Returns the endpoint.
 router.get(/* ... */)
 .tag('auth', 'restricted');
 ```
+
+securityScheme
+--------------
+
+`endpoint.securityScheme(type, [options], [description])`
```

Comment:
Not sure, but maybe find another name for "securityScheme", as _This method only affects the generated API documentation and has not other effect within the service itself._

Reply:
That's the name in OAS 3.0. The 2.0 name is securityDefinition (which is IMO even more ambiguous). It would be inconsistent with other documentation-only methods not to use the OAS name here.

There's also no way to build on this API to allow the user to provide an actual implementation as the API covers a wide range of schemes (e.g. basic auth, tokens in headers, tokens in URLs, or even OAuth 2.0).